{{cookiecutter.project_slug}}/README.md

```diff
@@ -23,8 +23,9 @@ See the [backend README](server/README.md)
 If not using Docker...
 See the [frontend README](client/README.md)
 
+
 ## Testing
 1. `pipenv install --dev`
-1. `pipenv run pytest server/{{cookiecutter.project_slug}}`
+1. `pipenv run pytest server/{{cookiecutter.project_name}}`
```

Comment:
pretty sure this should be the slug and not the name as the name would be capital and allow spaces

Reply:
Same issue here.